### PR TITLE
Travis service tests from tutorial

### PR DESCRIPTION
**Pull request by platan**
Service tests were introduced in https://github.com/badges/shields/pull/937 by @paulmelnikow. [The tutorial](https://github.com/badges/shields/blob/bde4d925f21fed2ddf87e9164e6c1ea7d3073d38/service-tests/README.md) is based on service tests of Travis CI, we do not have them in out codebase. Do you know why? I think it would be good idea to add them.

### CODE REVIEW

File: service-tests/travis.js [hunk 1]
```diff
@@ -4,9 +4,43 @@ const Joi = require('joi');
...
+  .get('/foo/bar.json')
+  .intercept(nock => nock('https://api.travis-ci.org')
+    .head('/foo/bar.svg')
+    .reply(200))
+  .expectJSON({ name: 'build', value: 'invalid' });
+
+t.create('connection error')
+  .get('/foo/bar.json')
+  .networkOff()
+  .expectJSON({ name: 'build', value: 'invalid' });
```
Comment (chris48s) [inline service-tests/travis.js]: usually we'd use 'inaccessible' for this case and 'invalid' when the response is malformed

**Comment by paulmelnikow** [comment]
The first version of the tutorial was written before the service tests. I never noticed they included code for tests that weren't added to the code base!

File: service-tests/travis.js (hunk 1 above)
Comment (paulmelnikow) [reply service-tests/travis.js]: Looks good to merge when this is addressed!

Comment (platan) [reply service-tests/travis.js]: Thanks @chris48s, great idea. @paulmelnikow done

**Comment by platan** [comment]
In the next PR I would like to use great `checkErrorResponse` function in the code handling Travis badge and in the tutorial.

**Pull request merged**